guides/v2.2/coding-standards/technical-guidelines.md

```diff
@@ -455,6 +455,8 @@ class View extends Template
 
 4.4. Plugins MUST be stateless.
 
+4.5. Plugins SHOULD NOT change a state of an intercepted object.
```

Comment:
What about extension attributes? This documentation recommends to use plugins to modify entity's state https://devdocs.magento.com/guides/v2.3/extension-dev-guide/extension_attributes/adding-attributes.html
The new rule contradicts with this documentation. What should be the replacement then?

Reply:
this rule does not contradict with the example.
In the example intercepted object is Repository.